app/main.go

```diff
@@ -61,8 +59,26 @@ func getIPAddress(r *http.Request) string {
 }
 
 func handlerPing(w http.ResponseWriter, r *http.Request) {
-	w.Write([]byte("pong"))
-	fmt.Println(time.Now(), getIPAddress(r), r.Method, r.RequestURI, r.UserAgent())
+	
+	requestBody, err := ioutil.ReadAll(r.Body)
+	n := rand.Intn(100)
+
+	select {
+		case <- time.After(500 * time.Millisecond):
+		if n <= 30 {
```

Comment:
This is great but it would be nice to have the probability provided by flag too. Right now it's always 30